packages/material-ui/src/TextareaAutosize/TextareaAutosize.js

```diff
@@ -63,15 +66,9 @@ const Textarea = React.forwardRef(function Textarea(props, ref) {
     }
     outerHeight = Math.max(outerHeight, singleRowHeight);
 
-    if (boxSizing === 'content-box') {
-      outerHeight -=
-        getStyleValue(computedStyle, 'padding-bottom') +
-        getStyleValue(computedStyle, 'padding-top');
-    } else if (boxSizing === 'border-box') {
-      outerHeight +=
-        getStyleValue(computedStyle, 'border-bottom-width') +
-        getStyleValue(computedStyle, 'border-top-width');
-    }
+    outerHeight +=
+      (boxSizing === 'border-box' ? padding : 0) + (boxSizing === 'border-box' ? border : 0);
```

Comment:
Could this be simplified to `boxSizing === 'border-box' ? padding + border : 0` or something close?

Reply:
Yes, great idea :).